### ISSUE TITLE
this libray  found not support Anglar13  any body Idia about that

### ISSUE THREAD

**Issue by rsurendraidexcel** [SortableJS/angular-sortablejs#260]
HI , i update  angular project,  somewher  my project use this libray . i found error  not support angular 12 and 13 version.  any Idia.

**Comment by neburnodrog** [comment]
There's already a pull request updating to Angular 13: https://github.com/SortableJS/ngx-sortablejs/pull/257
You could install the fork (https://github.com/Liquid-JS/nxt-sortablejs/tree/master) until the aforementioned pull request gets merged.

**Comment by 0xmosta** [comment]
We were able to update the lib to angular 13, if this one is no longer maintained we will publish ours and try to keep it updated with future angular/sortable updates

**Comment by Reixko** [comment]
@neburnodrog is that your fork ? would it be possible to add angular12 support ?
@Hekek if there would one great "fork"/version, that would be fully standalone and has support for angular 12, as well as 13 and then potentionally future versions, that would be wonderful and very appreciated :)

**Comment by jasonverber** [comment]
@smnbbrv or @gutenye or @carbonhe or @kamilchlebek there are some PRs like #246 and #257 where it would be great to see some progress. My team is looking for a replacement to our current drag and drop solution in Angular, but we are also moving to Angular 12+. If there aren't people currently in a place to review PRs, etc. I'm sure there are folks in the community, my team among them, who would be ready to step forward.

**Comment by BenRacicot** [comment]
@jasonverber and @Hekek perhaps we could all begin a new project based on this one?

**Comment by jasonverber** [comment]
@BenRacicot we should probably also talk to [Anikethana](https://github.com/Anikethana) and [@hitko](https://github.com/HitkoDev) who opened #246 and #257 

Unless somebody like @andresin87 (who actually responded in another thread) can get something moving in this repo, or at least get the attention of somebody who can do something.

**Comment by spierala** [comment]
Yeah, very sad to see ngx sortablejs not being maintained. There are even community contributions to make it work again with ng12 and ng13. Why not merge that MRs (#246 and #257)?

**Comment by COBRASoft5** [comment]
Also interested to see this package continue and updated. There are some important parts missing like canceling a drag & drop operation on drop.

**Comment by Klapik** [comment]
Any news?

**Comment by hadley8899** [comment]
Bump on this

**Comment by BenRacicot** [comment]
Still no movement? Any good forks being maintained?
Looks like https://github.com/mrphelko/ngx-sortablejs but I cannot see a way to contact @MrPhelko

Personally I'd be willing to help and also financially support this project. 
**We need a maintained drag and drop package that is not the Angular CDK.**

**Comment by suhailkc** [comment]
Found an alternative solution to support SortableJs for Angular 18+:
[https://www.npmjs.com/package/nxt-sortablejs](https://www.npmjs.com/package/nxt-sortablejs)
